packages/autorest.python/samples/specification/azure_key_credential/generated/azure/key/credential/sample/_patch.py

```diff
@@ -2,6 +2,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
+
```

Comment:
When regenerate in DevBox, there is always empty line added. But it doesn't influence CI and after this commit,  there is no more new empty line added when regenerate again in DevBox, so I will keep them in this PR.